src/main/java/eu/eutampieri/catacombs/game/Main.java

```diff
@@ -17,7 +17,7 @@ public static void main(final String[] args) {
         gc.setGameHeight(480);
 
         gc.setScaling(true);
-        gc.setResizeable(true);
+        gc.setResizeable(false);
```

Comment:
Why?

Reply:
Because it does not work. At least if it is disabled, my wm does not try to resize it and i can see the stuff properly
![cacca](https://user-images.githubusercontent.com/22148361/114914297-6aa1d700-9e22-11eb-8db2-f4d8c0e279ca.png)

Reply:
Nvm, I was curious because this is counterintuitive